### ISSUE TITLE
Enabling TUI build disables GUI build; should allow building both

### ISSUE THREAD

**Issue by salfter** [skyjake/lagrange#533]
Someone might want to have both versions available for different circumstances, but the build system only allows one or the other.  Providing two separate ebuilds (I'm on Gentoo) didn't work either, as they try to install the same libraries, which causes conflicts.

Perhaps an additional cmake variable, ENABLE_GUI, that defaults to true and can be set to false if a TUI-only build is desired would do the trick.

**Comment by skyjake** [comment]
I agree the current setup is not perfect. `clagrange` remains a bit experimental and there may still be substantial changes. Ultimately, it should be possible to expand the main CMakeLists to allow both GUI and TUI builds in parallel, although it will require adding [SEALCurses](https://git.skyjake.fi/skyjake/sealcurses) as another external library/submodule under _lib_.
